Scripts/gen_Mix.sh

```diff
@@ -250,6 +250,8 @@ fi
 if [ "${PRIMARY}" == "NoPrimary" ]; then
   echo '#include "Production/JobConfig/mixing/NoPrimary.fcl"' >> mix.fcl
 fi
+# Override dts filters conditioned on primary
+echo "#include \"Production/JobConfig/mixing/filters/${PRIMARY}.fcl\"" >> mix.fcl
```

Comment:
I see this is where you use the files defined above.
Since gen_Mix.sh is the 'master script' in the campaign I suggest to let it write a (transient) filter configuration directly into the main job fcl for the cases that needed it, and leave it out for the cases that don't. That avoids needing a large number of empty files.

Reply:
I'd argue for keeping a static-fcl-based workflow, even if constrained to obviate any empty files. I find all of the "master" campaign scripts to be a bit hard to parse, but the bulk of their output is constructed by stitching together upstream static fcl stubs. The current implementation keeps with that, so that a domain expert implementing / changing the filter for a given primary need only populate the corresponding fcl in `.../mixing/filters`. That does require that empty files exist for non-overlapping primaries (conversion electrons, etc.), but it relieves that implementer from having to touch or understand the campaign script.

The far alternative is to instead embed the actual filter implementations into a series of if-else clauses in the `gen_Mix.sh` itself, which is how I interpret your suggestion. The downside I see with that is that it will require continued interaction with / iteration of that script as more primaries are correctly implemented.

The middle ground would be either a) explicitly if-elseing the inclusion of filter fcl stubs conditioned on the primary, or b) extending the logic from "include the stub for this primary" to "include the stub for this primary if that stub exists", so that the empty files can be removed. (a) would require continued changes the script, and (b) allows for mistakes if someone doesn't test prudently, but we can probably survive that. What do you think of (b)?